dev/io.openliberty.microprofile.telemetry.1.0.internal/resources/io/openliberty/microprofile/telemetry/internal/resources/MPTelemetry.nlsprops

```diff
@@ -29,3 +29,7 @@
 CWMOT5100.tracing.is.disabled=CWMOT5100I: The MicroProfile Telemetry Tracing feature is enabled but not configured to generate traces for the {0} application.
 CWMOT5100.tracing.is.disabled.explanation=The MicroProfile Telemetry Tracing feature needs to be configured to create traces and export them to a tracing system.
 CWMOT5100.tracing.is.disabled.useraction=To enable tracing, specify otel.sdk.disabled=false in any of the configuration sources accessible through MicroProfile Config. Further configuration might be required to allow the MicroProfile Telemetry Tracing feature to export traces to your tracing system.
+
+CEMOT5101.telemetry.error=CEMOT5101: An exception was thrown from MicroProfile Telemetry. See the assocaited exception: {0}.
```

Comment:
Suggestion: Remove "See the assocaited exception:"

If you want to include the sentence:
- Either remove `assocaited` or change it to `associated`.
- Consider replacing "exception" with "error". We try to use "error" rather than "exception" to simplify message text. But, if "exception" is needed to clarify what was thrown, then keep "exception".

Reply:
Hello @helyarp thank you for the suggestions. 

Happy to replace "exception" with "error" or if you prefer you can completely rewrite the message to whatever you like.